### PR TITLE
Add stringify function (Jairus Tanaka)

### PR DESCRIPTION
**Pull request by JairusSW**
Here's the stringify function!
I ran a bench (with benchmark) and here's the results.

I'm using an Intel i3 that's overclocked to 3.9ghz. 16 GB RAM.
```
TinySonic Stringify x 7,089,854 ops/sec ±9.23% (82 runs sampled)
Jsonic Stringify x 1,565,541 ops/sec ±1.01% (88 runs sampled)
JSON Stringify x 2,562,274 ops/sec ±0.91% (87 runs sampled)
TinySonic Parse x 2,067,367 ops/sec ±2.39% (88 runs sampled)
Jsonic Parse x 159,366 ops/sec ±9.79% (91 runs sampled)
JSON Parse x 1,434,668 ops/sec ±13.47% (81 runs sampled)
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "flat",
     "object"
   ],
-  "author": "Matteo Collina <hello@matteocollina.com>",
+  "author": "Matteo Collina <hello@matteocollina.com> and Jairus Tanaka <jairus.v.tanaka@outlook.com>",
```
Comment (mcollina) [inline package.json:20]: Please revert this, maybe add yourself in the collaborators

File: tinysonic.js [hunk 2]
```diff
@@ -80,4 +80,100 @@ function asValue (str) {
...
+  randomData: {
+    hello: 'world'
+  }
+})
+
+console.log(encoded)
+
+const decoded = parse(encoded)
+
+console.log(decoded)*/
```
Comment (mcollina) [inline tinysonic.js:179]: please remove these comments

File: package.json (hunk 1 above)
Comment (JairusSW) [reply package.json:20]: Sure. Do you want to edit it, or should I submit another pull?

**Review by mcollina — CHANGES_REQUESTED** [review]
please add a test

**Comment by JairusSW** [comment]
@mcollina , a test in test.js?

**Comment by mcollina** [comment]
Yes exactly. This project needs a bit of maintance as well.. for example dropping travis and moving it to Github actions.

**Comment by JairusSW** [comment]
Okay, gotcha! I'll be done in a while.

**Comment by JairusSW** [comment]
@mcollina , okay! Its in the forked repo. https://github.com/JairusSW/tinysonic

**Review by mcollina — COMMENTED** [review]
Thanks for the good work! It seems linting is failing, could you take a look?

**Comment by JairusSW** [comment]
Sure. Using eslint?

**Comment by mcollina** [comment]
This uses standard

**Comment by JairusSW** [comment]
Awesome. All lints passed. All tests too. 😄 Its all in the fork.

**Comment by mcollina** [comment]
Why did you close the PR?

**Comment by JairusSW** [comment]
I don't think I did that... Idk.

**Comment by JairusSW** [comment]
Ah, I had to force-push. Let me refork. Hold on.

**Comment by JairusSW** [comment]
A wierd error popped up. cat is not definied

**Pull request closed without merging**